cmd/protoc-gen-gorums/dev/zorums_ordered_gorums.pb.go

```diff
@@ -460,10 +460,120 @@ func (s *GorumsServer) RegisterOrderingComboHandler(handler OrderingComboHandler
 		if err != nil {
 			return new(ordering.Message)
 		}
-		return &ordering.Message{Data: data, MethodID: orderingComboMethodID}
+		return &ordering.Message{ID: in.ID, Data: data, MethodID: orderingComboMethodID}
 	})
 }
 
+// OrderingConcurrent is a quorum call invoked on all nodes in configuration c,
+// with the same argument in, and returns a combined result.
+func (c *Configuration) OrderingConcurrent(ctx context.Context, in *Request) (resp *Response, err error) {
+	var ti traceInfo
+	if c.mgr.opts.trace {
+		ti.Trace = trace.New("gorums."+c.tstring()+".Sent", "OrderingConcurrent")
+		defer ti.Finish()
+
+		ti.firstLine.cid = c.id
+		if deadline, ok := ctx.Deadline(); ok {
+			ti.firstLine.deadline = time.Until(deadline)
+		}
+		ti.LazyLog(&ti.firstLine, false)
+		ti.LazyLog(&payload{sent: true, msg: in}, false)
+
+		defer func() {
+			ti.LazyLog(&qcresult{reply: resp, err: err}, false)
+			if err != nil {
+				ti.SetError()
+			}
+		}()
+	}
+
+	// get the ID which will be used to return the correct responses for a request
+	msgID := c.mgr.nextMsgID()
+
+	// set up a channel to collect replies
+	replies := make(chan *orderingResult, c.n)
+	c.mgr.putChan(msgID, replies)
+
+	// remove the replies channel when we are done
+	defer c.mgr.deleteChan(msgID)
+
+	data, err := proto.MarshalOptions{AllowPartial: true, Deterministic: true}.Marshal(in)
```

Comment:
Same for these marshal options (and unmarshal) below. Maybe these could be using some globally defined option variable?